rs/nns/governance/canister/canister.rs

```diff
@@ -149,8 +150,38 @@ fn set_governance(gov: Governance) {
         .expect("Error initializing the governance canister.");
 }
 
+// Seeding interval seeks to find a balance between the need for rng secrecy, and
+// avoiding the overhead of frequent reseeding.
+const SEEDING_INTERVAL: Duration = Duration::from_secs(3600);
+const RETRY_SEEDING_INTERVAL: Duration = Duration::from_secs(30);
+
+fn schedule_seeding(duration: Duration) {
```

Comment:
@andrew-lee-work Could you re-review this function?

Reply:
LGTM with the trap removed from the error case. Of course it could still trap for other reasons, but this reduces the possibility.